docs/GeneralPurposeAnnotationProcessors.md

```diff
@@ -54,8 +56,36 @@ Before
 After
 ```
 
+## Symfony Expression Language
+**Description:** Uses [Symfony Expression Language][Symfony EL] to generate the replacement.
```

Comment:
> [Symfony Expression Language][Symfony EL]

neat!

Reply:
Yeah. I almost always try to use link references. It makes it easier to read when viewing raw markdown and easier to change or update.